packages/widget/src/components/Modal.tsx

```diff
@@ -84,10 +85,11 @@ export const createModal = <T extends ModalProps>(
   const Component = component;
 
   const WrappedComponent = (props: T) => {
-    const [, setError] = useAtom(errorAtom);
+    const setError = useSetAtom(errorAtom);
+    const theme = useAtomValue(themeAtom);
 
     return (
-      <Modal {...props}>
+      <Modal {...props} theme={theme}>
```

Comment:
once we pass `theme` from `themeAtom` here we can use `useTheme()` from `styled-components` as usual (which is what all of the modals are currently doing)